bin/packages/validate-typescript-version.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+const tscDetectedVersion = require( 'typescript' ).version;
+const tscDependencyVersion = require( '../../package.json' ).devDependencies
+	.typescript;
+
+/* eslint-disable no-console */
+
+if ( tscDependencyVersion !== tscDetectedVersion ) {
+	console.error(
+		[
+			'TypeScript dependency out of date.',
+			'\tDetected: %o',
+			'\tRequired: %o',
```

Comment:
Huh, I wasn't aware this usage is a simple way for adding a splash of color to the output.

![image](https://user-images.githubusercontent.com/1779930/77781353-cf2da080-702b-11ea-839a-817cbf0af844.png)